tox.ini

```diff
@@ -20,8 +20,8 @@ envlist =
     pep8
     py38-openssl
     py39-dnspython1
-    py{27,35,36,py2,py3}-epolls
-    py{37,38,39,310}-{selects,poll,epolls}
+    pypy3-epolls
+    py{38,39,310}-{selects,poll,epolls}
```

Comment:
Not sure why the deps of below `testenv` manage Python 3.11 and why the above `envlist`, apparently, not. As the semantic of these changes is to drop EOLed Python releases, this patch LGTM.